iaso/tests/api/test_profiles_bulk_update.py

```diff
@@ -20,124 +22,158 @@ def saveUserProfile(user):
+            language="en",
+            user_roles=[cls.user_role],
+        )
+        saveUserProfile(cls.user_with_user_role1)
+
+        cls.user_admin_account2 = cls.create_user_with_profile(
+            username="user_admin_account2",
+            account=cls.account2,
             permissions=[permission._USERS_ADMIN, permission._DATA_TASKS],
             language="en",
         )
-        saveUserProfile(cls.raccoon)
-        cls.root = cls.create_user_with_profile(
-            username="root", account=marvel, permissions=[permission._USERS_ADMIN], org_units=None, language="en"
+        saveUserProfile(cls.user_admin_account2)
+        cls.user_admin_no_task_account2 = cls.create_user_with_profile(
+            username="user_admin_no_task_account2",
+            account=cls.account2,
+            permissions=[permission._USERS_ADMIN],
+            org_units=None,
+            language="en",
         )
-        saveUserProfile(cls.root)
-        cls.wolverine = cls.create_user_with_profile(
-            username="wolverine",
-            account=zelda,
+        saveUserProfile(cls.user_admin_no_task_account2)
+        cls.user_admin_account3 = cls.create_user_with_profile(
+            username="user_admin_account3",
+            account=cls.account3,
             permissions=[permission._USERS_ADMIN, permission._DATA_TASKS],
             org_units=None,
             language="en",
         )
-        saveUserProfile(cls.wolverine)
+        saveUserProfile(cls.user_admin_account3)
 
         cls.user_with_no_users_permission = cls.create_user_with_profile(
             username="userNoUsersPermission",
-            account=star_wars,
+            account=cls.account1,
             permissions=["iaso_data_tasks"],
             language="en",
         )
         saveUserProfile(cls.user_with_no_users_permission)
 
         cls.add_users_team_launcher_1 = cls.create_user_with_profile(
             username="add_users_team_launcher_1",
-            account=star_wars,
+            account=cls.account1,
             permissions=[permission._USERS_ADMIN, permission._DATA_TASKS, permission._TEAMS],
             language="en",
         )
         saveUserProfile(cls.add_users_team_launcher_1)
 
         cls.team_1 = m.Team.objects.create(
-            name="Team 1", manager=cls.yoda, project=cls.project, type=TeamType.TEAM_OF_USERS
+            name="Team 1", manager=cls.user_admin, project=cls.project, type=TeamType.TEAM_OF_USERS
+        )
+        cls.team_2 = m.Team.objects.create(
+            name="Team 2", manager=cls.user_admin, project=cls.project, type=TeamType.TEAM_OF_USERS
+        )
+        cls.team_3 = m.Team.objects.create(
+            name="Team 3", manager=cls.user_admin, project=cls.project, type=TeamType.TEAM_OF_USERS
+        )
+        cls.team_of_teams = m.Team.objects.create(
+            name="Team of Teams", manager=cls.user_admin, project=cls.project, type=TeamType.TEAM_OF_TEAMS
```

Comment:
How does this work? Where are the teams/users added to these teams?

Reply:
I assign some of the existing users in the `test_bulk_update_with_teams_filters` test